.github/workflows/test.yml

```diff
@@ -33,6 +33,43 @@ jobs:
             inst/validation
           path: templates
 
+      - name: Overwrite quality control scripts with dv.templates ones 🧹
+        shell: bash
+        run: |
+          TEMPLATE_DIR="templates/inst/validation"
+          REPO_DIR="inst/validation"
+
+          if test ! -d "${REPO_DIR}"; then
+            echo "### Quality Control documentation is not present"
+            echo "### Include quality control documentation or skip it by creating following file 'inst/validation/skip_qc'"
+            exit 1
+          fi
+
+          if test -f "${REPO_DIR}/skip_qc"; then
+            echo "skip_qc file found skipping qc documentation"
```

Comment:
we could have a variable switch instead of a file with a specific name here

Reply:
See other comment